core/templates/dev/head/domain/exploration/FallbackObjectFactory.js

```diff
@@ -17,7 +17,9 @@
  * domain objects.
  */
 
-oppia.factory('FallbackObjectFactory', [function() {
+oppia.factory('FallbackObjectFactory', [
+    'OutcomeObjectFactory',
```

Comment:
You can deindent this by 2; the following line is still an element of the same list.